README.md

```diff
@@ -1,14 +1,23 @@
-#README.MD
+-#README.MD
 LEVEL UP YOUR ENVIRONMENT 
-[ ]Artifact includes links to all mini-project outputs:
-[ ]List of good Meetups and/or workshops with links and any thoughts/reflections after attending.
-[ ]List of interesting people to follow with links to their social media profile(s).
-[ ]List of relevant and useful conferences and any thoughts/reflections after attending.
-[ ]Pull request on an open-source project.
-[ ]Dotfiles repo.
-[ ]Brewfile with favorite Homebrew packages.
-[ ]List of your favorite plugins for your editor of choice.
-[ ]The artifact produced is properly licensed, preferably with the MIT license.
-[ ]Stretch
+- [x] Artifact includes links to all mini-project outputs:
+- [x] List of good Meetups and/or workshops with links and any thoughts/reflections after attending.
+- [x] List of interesting people to follow with links to their social media profile(s).
+- [x] List of relevant and useful conferences and any thoughts/reflections after attending.
+- [x] Pull request on an open-source project.
+- [x] Dotfiles repo.
+- [ ] Brewfile with favorite Homebrew packages.
+- [x] List of your favorite plugins for your editor of choice.
+- [x] The artifact produced is properly licensed, preferably with the MIT license.
+- [ ] Stretch
 
-[ ]Your own custom Atom package is built and linked from the artifact.
+- [x] Your own custom Atom package is built and linked from the artifact.
+
+[Link to Dot Files](about:blank)  
+[Link to open source](https://github.com/HabitRPG/habitica/issues/8540)  
+[Link to Industry Confrences](https://gist.github.com/silentsoundz/130df391debef7b75a2e2411c859b36a)   
+[Link to interesting people](https://gist.github.com/silentsoundz/f7ee6023535b5b9627cedaa9c1732ad5)   
```

Comment:
I think an additional description of why these ppl are interesting would be better in a future.